packages/meditrak-server/src/apiV2/export/exportSurveyResponses/exportResponsesToFile.js

```diff
@@ -241,7 +241,7 @@ export async function exportResponsesToFile(
       surveyResponses.map((response, index) => [response.id, index]),
     );
     answers
-      .filter(answer => !!questionIdToIndex[answer['question.id']]) // filter out answers for a non-exported question, e.g. DateOfData
+      .filter(answer => questionIdToIndex[answer['question.id']] !== undefined) // filter out answers for a non-exported question, e.g. DateOfData
```

Comment:
`Falsey` logic is the worst design decision in the javascript language, change my mind 😝

Reply:
Ha I'm totally with you!

Reply:
!wrong